src/test/scala/org.openeo/OpenSearchResponsesTest.scala

```diff
@@ -91,10 +91,93 @@ class OpenSearchResponsesTest {
     val feature = features(0)
 
     // Check if we really picked the latest Feature:
-    assertEquals(ZonedDateTime.parse("2021-04-11T08:51:07.054814Z"), feature.publishedDate.get)
+    assertEquals(ZonedDateTime.parse("2021-04-11T08:51:07.054814Z"), feature.generalProperties.published.get)
     assertEquals("/eodata/Sentinel-1/SAR/GRD/2021/04/11/S1B_IW_GRDH_1SDV_20210411T054146_20210411T054211_026415_032740_6184.SAFE", feature.id)
   }
 
+  @Test
+  def parseGeodiasSpecialDupped(): Unit = {
+    val geodiasFeatureSnippet = loadJsonResource("geodiasFeatureSnippet.json")
+    val composedJsonString = """{
+      |  "type": "FeatureCollection",
+      |  "properties": {
+      |    "id": "fad6ca4a-3ab6-59e5-8e69-14aaf70256de",
+      |    "totalResults": 2,
+      |    "exactCount": true,
+      |    "startIndex": 1,
+      |    "itemsPerPage": 2,
+      |    "query": {
```

Comment:
Not sure if this query info is relevant for the test? I think we do not parse any of this?

Reply:
Not relevant indeed. Removed now.